ci-operator/step-registry/ipi/conf/aws/metadata-service-auth/ipi-conf-aws-metadata-service-auth-ref.yaml

```diff
@@ -0,0 +1,19 @@
+ref:
+  as: ipi-conf-aws-metadata-service-auth
+  commands: ipi-conf-aws-metadata-service-auth-commands.sh
+  from_image:
+    namespace: ocp
+    name: "4.5"
```

Comment:
4.5?

Reply:
Dunno why, but all steps all around there have 4.5. Just made it the same, since i don't understand the reasons.

Reply:
Ok, good enough reasoning for me